simple-example/README.md

```diff
@@ -35,8 +35,9 @@ There are five files here:
 - _[work.py](./src/work.py)_ - This contains the main code for the model. Here, as an example, it is a
   simple Fibonacci generator
 - _[Dockerfile](./Dockerfile)_ - Builds the container that will be run by DAFNI
-- _[model_definition.yaml](./model_definition.yaml)_ - Holds details of the model needed by the DAFNI ingest system.
-- _README.md_ - This helpful file.
+- _[model_definition.yaml](./model_definition.yaml)_ - A machine readable file with defining the model.
```

Comment:
There is extra/missing word in this line, not sure which way you want to take it!